python/pyproject.toml

```diff
@@ -41,7 +41,7 @@ classifiers = [
 test = [
     "cloudpickle",
     "cudf==24.2.*",
-    "cupy",
+    "cupy-cuda11x>=12.0.0",
```

Comment:
We're inconsistent in how we handle cudf/cupy fallback. This is a problem across basically all of RAPIDS, so it'd be good for us to unify our approaches.

If we need `cupy-cuda11x` here, we would probably want `cudf-cu11` as well. For all RAPIDS packages we've been falling back to the "conda name" but we've been falling back to the "CUDA 11 pip name" for cupy. We should do one of:
- use `cudf` and `cupy`
- use `cudf-cu11` and `cupy-cuda11x`

Reply:
I'm using the "conda name" here because it works automatically when building the repos from source.

Reply:
Gotcha. I wondered about that. We can leave this as-is, then.